src/MusicWheel.cpp

```diff
@@ -799,7 +802,7 @@ void MusicWheel::BuildWheelItemDatas( std::vector<MusicWheelItemData *> &arrayWh
 								unsigned j;
 								for( j=i; j < arraySongs.size(); j++ )
 								{
-									if( SongUtil::GetSectionNameFromSongAndSort( arraySongs[j], so ) != sThisSection )
+									if( SONGMAN->GetGroup(arraySongs[j])->GetGroupName() != sThisSection )

```

Comment:
Can we null check this in case GetGroup returns null. Log a warning if so. basically everywhere in this PR that chains ->GetGroup->getsomething.

It'll prevent any crashes in case there are any.